### ISSUE TITLE
add currency in cospend

### ISSUE THREAD

**Issue by kds69** [mayflower/PayForMe#72]
Since projects in Cospend have currencies now, would be instrumental to have currency (default one and override) at bill level in PayForMe.
Makes a lot of sense as this app is very useful when travelling abroad with friends.

**Comment by Hustenbonbon** [comment]
good point, but as I am not working in app development anymore and not actively using PayForMe/Cospend myself, I personally will only spend time in keeping the current functionality intact as good as I can.
